packages/endpoint/1.2.2-dev.0/changelog.yml

```diff
@@ -0,0 +1,86 @@
+##
+## This file documents the package changelog
+##
+
+
+- version: 1.2.2
+  changes:
+    - description: malware_signature.primary.matches should be keyword
+      type: bugfix
+      link: https://github.com/elastic/endpoint-package/pull/212
```

Comment:
changelog entry matching this release version (once out of dev into prod release status)